web/src/main/java/org/mskcc/cbio/portal/swagger/ApiServiceSwaggerConfig.java

```diff
@@ -17,7 +17,7 @@
 public class ApiServiceSwaggerConfig {
     @Bean
     public Docket api() {
-        return new Docket(DocumentationType.SWAGGER_2)
+        return new Docket(DocumentationType.SWAGGER_2).groupName("legacy")
```

Comment:
Is this a change that has been discussed? I believe elimination of the legacy api controller is desired, so it may be moot how we group it in swagger anyway.  cc: @n1zea144